app/models/agents/chattermill_response_agent.rb

```diff
@@ -170,24 +169,28 @@ def normalize_response_headers(headers)
       }
     end
 
-    def post_url(event = Event.new)
+    def request_url(event = Event.new)
       event_options = interpolated(event.payload)
       protocol = Rails.env.production? ? 'https' : 'http'
       domain = DOMAINS[Rails.env.to_sym]
       host = "#{event_options['organization_subdomain']}.#{domain}"
+      "#{protocol}://#{host}#{API_ENDPOINT}/#{event.payload.dig('data', 'id')}"
+    end
 
-      "#{protocol}://#{host}#{API_ENDPOINT}"
+    def has_id?(event)
+      event.payload['data'] && event.payload['data']['id']
     end
 
     def auth_header
       { "Authorization" => "Bearer #{ENV['CHATTERMILL_AUTH_TOKEN']}" }
     end
 
     def handle(data, event = Event.new, headers)
-      url = post_url(event)
+      url = request_url(event)
       headers['Content-Type'] = 'application/json; charset=utf-8'
       body = data.to_json
-      response = faraday.run_request(http_method.to_sym, url, body, headers)
+      method = http_method(event)
+      response = faraday.run_request(method, url, body, headers)
       send_slack_notification(response, event) unless response.status == 201
```

Comment:
@OpakAlex One last little thing 😅, now that we're updating, status should be `200` or `201`